sky/spot/controller.py

```diff
@@ -177,13 +182,26 @@ def run(self):
         except exceptions.ResourcesUnavailableError as e:
             logger.error(f'{common_utils.class_fullname(e.__class__)}: '
```

Comment:
Document this except will only be entered iff retry_until_up is false and we've maxed out the RETRY_COUNT.